src/api/ingest/bleaching_serializer.py

```diff
@@ -57,6 +57,15 @@ def benthic_attributes_choices():
     ]
 
 
+class BleachingCSVListSerializer(CollectRecordCSVListSerializer):
+    def create_hook(self, grouped_records):
```

Comment:
Couldn't you just override `group_records`, and then avoid `output = self.create_hook(output)` ?